apps/wallet-connect/src/App.tsx

```diff
@@ -62,19 +78,20 @@ const App = () => {
             <StyledCard>
               {connectionStatus === CONNECTION_STATUS.DISCONNECTED && (
                 <Disconnected>
-                  <WalletConnectField client={wcClientData} onConnect={data => wcConnect(data)} />
+                  <WalletConnectField wcClientData={wcClientData} wcConnect={wcConnect} />
```

Comment:
Why renaming the props here? the `onXXX` is the code convention in the other components

Reply:
done!